s3-source-connector/src/main/java/io/aiven/kafka/connect/s3/source/utils/FileReader.java

```diff
@@ -70,17 +72,30 @@ List<S3ObjectSummary> fetchObjectSummaries(final AmazonS3 s3Client) throws IOExc
                     .stream()
                     .filter(objectSummary -> objectSummary.getSize() > 0)
                     .filter(objectSummary -> !failedObjectKeys.contains(objectSummary.getKey()))
+                    // .filter(objectSummary -> !inProcessObjectKeys.contains(objectSummary.getKey()))
                     .collect(Collectors.toList());
 
             allSummaries.addAll(filteredSummaries); // Add the filtered summaries to the main list
 
+            // TO BE REMOVED before release
```

Comment:
This good for now because we have the TODO but we could also consider something like.
Might be worth putting a if(LOGGER.isDebugEnabled()) {
 allSummaries.forEach(objSummary -> LOGGER.info("Objects to be processed {} ", objSummary.getKey()));
}

If and only if this could be valuable for debugging issues in the future.